src/core/ui/components/drop-in/drop-down.tsx

```diff
@@ -0,0 +1,414 @@
+import { useUser } from '../../context/user';
+import { useMemo, useState } from 'react';
+import { useSlashAuth } from '../../../context/legacy-slashauth';
+import { ScaleLoader } from 'react-spinners';
+
+export const DropDown = () => {
```

Comment:
Does clicking otuside of of this dropdown close it?

Reply:
updated